com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/util/DeployedModuleCalculator.java

```diff
@@ -0,0 +1,11 @@
+package com.sap.cloud.lm.sl.cf.process.util;
+
+import java.util.List;
+
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMtaModule;
+
+public interface DeployedModuleCalculator {
```

Comment:
Calculator is used for calculating one thing from two or more other things.
Your class is taking one set of modules and it is returning the same set.